test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua

```diff
@@ -0,0 +1,136 @@
+--! @parameters: NO
+--]]
+local function update_sdl_preloaded_pt_json()
+	local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+	local file = io.open(pathToFile, "r")
+	local json_data = file:read("*all") -- may be abbreviated to "*a";
+	file:close()
+				
+	local json = require("modules/json")
+	
+	local data = json.decode(json_data)
+	for k in pairs(data.policy_table.functional_groupings) do
+		if (data.policy_table.functional_groupings[k].rpcs == nil) then
+			data.policy_table.functional_groupings[k] = nil
+		end
+	end
+
+	data.policy_table.app_policies["0000001"] = {
+		keep_context = false,
+		steal_focus = false,
+		priority = "NONE",
+		default_hmi = "NONE",
+		groups = {"Base-4"}
+	}
+	data.policy_table.app_policies["0000001"].AppHMIType = {"NAVIGATION"}
+				
+	data = json.encode(data)
+	file = io.open(pathToFile, "w")
+	file:write(data)
+	file:close()
+end
+
+--[[ General Precondition before ATF start ]]
+commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+update_sdl_preloaded_pt_json()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+
+--TODO(istoimenova): shall be removed when issue: "ATF does not stop HB timers by closing session and connection" is fixed
+config.defaultProtocolVersion = 2
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}	
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_InitHMI_OnReady()
+	testCasesForRAI.InitHMI_onReady_without_UI_GetCapabilities(self)
+	local event = events.Event()
+	event.level = 2
+	event.matches = function(_, data) return data.method == "UI.GetCapabilities" end
+
+	EXPECT_HMIEVENT(event, "UI.GetCapabilities")
+	-- HMI does not reply to UI.GetCapabilities
+end
+
+function Test:Precondition_connectMobile()
+	self:connectMobile()
+end
+
+function Test:Precondition_StartSession()
+	self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+	self.mobileSession:StartService(7)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_RAI_steeringWheelLocation()
+	local CorIdRegister = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+		
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_RESPONSE(CorIdRegister, { success = true, resultCode = "WARNINGS", hmiCapabilities = { steeringWheelLocation = value_steering_wheel_location } })
+	EXPECT_NOTIFICATION("OnHMIStatus", { systemContext = "MAIN", hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE"})
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+
+function Test.Postcondition_Restore_config_files()
+	commonPreconditions:RestoreFile("hmi_capabilities.json")
```

Comment:
@istoimenova 
You have remove call of function 
`commonPreconditions:RestoreFile("sdl_preloaded_pt.json").`
Please get it back, it is used necessarily and remove function `commonPreconditions:RestoreFile("hmi_capabilities.json")` instead.